loops_labs.js

```diff
@@ -0,0 +1,103 @@
+//Write a while loop and a for loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
```

Comment:
Good demonstration of understanding for for & while loops. Good indentation for codes and logics made sense. Keep up the good work!